spec/unit/blockscore/collection_spec.rb

```diff
@@ -1,178 +1,48 @@
 module BlockScore
   RSpec.describe Collection do
-    let(:parent) { create(:fake_resource) }
-    let(:member_class) { FakeResource }
-    let(:collection) { Collection.new(parent, member_class) }
+    let(:person) { create(:person) }
+    subject(:question_sets) { person.question_sets }
 
-    before do
-      Util::PLURAL_LOOKUP['fake_resource'] = 'fake_resources'
-      allow(member_class).to receive(:create) { create(:fake_member, parent_id: parent.id) }
-    end
-
-    describe '#all' do
-      subject { collection.all  }
+    it { expect(question_sets.empty?).to be true }
+    it { expect(question_sets.class).to be BlockScore::Collection }
```

Comment:
I don't feel strongly either way here, but since the following line

```ruby
it { expect(question_sets.class).to be BlockScore::Collection }
```

shows up 5 times in this file my gut reaction would be to reach for `shared_context`